### PR TITLE
Update README.md to include setup instructions

### PR DESCRIPTION
**Pull request by Gabri1998**
### :ballot_box_with_check: Definition of Done checklist
- [x] Meaningful title and valuable description in pull request
- [x] Code change tested manually if not covered by automated tests
- [x] Pull request linked to one or more issues/stories
- [x] Acceptance criteria of the linked stories satisfied
- [x] Code change comprehended by those who approved it
- [x] State of target branch maintained or improved after the code change

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,7 +33,109 @@ Pocket Parking will use the car's sensors to detect obstacles and avoid collisio
 **MqttClient.java**
 > The MqttClient.java file is mainly responsible for setting up the MQTT protocol for the Android application. In this file, there are methods that connect and disconnect the mobile application to the broker. Also methods for subscribing, unsubscribing and publishing from the mobile application is implemented in this part of the system.
 
-***
+**Setup Instruction**
+
+-To start testing the pocket car parking product on your computer:-
+ 1st :
+You need to download SMCE, Mosquitto and Android Studio
+Android Studio
```
Comment (arrestoteles) [inline README.md:41]: This is shown twice in the text. You can remove this "Android Studio" ✏️

File: README.md [hunk 2]
```diff
@@ -33,7 +33,109 @@ Pocket Parking will use the car's sensors to detect obstacles and avoid collisio
 **MqttClient.java**
 > The MqttClient.java file is mainly responsible for setting up the MQTT protocol for the Android application. In this file, there are methods that connect and disconnect the mobile application to the broker. Also methods for subscribing, unsubscribing and publishing from the mobile application is implemented in this part of the system.
 
-***
+**Setup Instruction**
+
+-To start testing the pocket car parking product on your computer:-
```
Comment (arrestoteles) [inline README.md:38]: Change "pocket car parking" into "pocket parking..."

File: README.md [hunk 3]
```diff
@@ -33,7 +33,109 @@ Pocket Parking will use the car's sensors to detect obstacles and avoid collisio
...
+
+<img src="https://user-images.githubusercontent.com/90387423/170671081-0825909c-f83c-4065-baa6-2ba2adaec0f8.png" width="400">
+
+
+
+
+3rd: 
+Find a file in the repo at location group-02/mod/ppmod.pck, copy it and paste it in the mods directory of the SMCE-gd app (SMCE/mods), for it to register the terrain and vehicle. The location is different based on your operating system, use the relevant location from the list below.  
+  
+For Windows: C:/Users/admin/AppData/Roaming/Godot/app_userdata/SMCE/mods
```
Comment (arrestoteles) [inline README.md:76]: In general - just clarify if this applied to all major OS's or if there are any differences 💻

File: README.md [hunk 4]
```diff
@@ -33,7 +33,109 @@ Pocket Parking will use the car's sensors to detect obstacles and avoid collisio
...
+
+
+
+
+
+
+
+
+
+6th:
```
Comment (arrestoteles) [inline README.md:134]: Good job with the instructions! ⭐

**Review by arrestoteles — CHANGES_REQUESTED** [review]
Some small notations - otherwise good job boys! 👍

**Review by arrestoteles — APPROVED** [review]

**Comment by Gabri1998** [comment]
A user now would be able to install and  setup the product to test through the instructions in the Readme

**Pull request merged**